index.js

```diff
@@ -119,8 +120,11 @@ function createHarness (conf_) {
                 if (!r.ok && typeof r !== 'string') test._exitCode = 1
             });
         })(t);
-        
+    
         results.push(t);
+        if (t.number === onlyTestNumber)
+            results.only(onlyTestNumber);
```

Comment:
please add curly braces around all if blocks, in this case for consistency with lines 107-109.